### PR TITLE
feat: e2e rule to have assignee comment on `test.fixme`

### PR DESCRIPTION
**Pull request by harshal015**
## Description
app.clickup.com
Brief description of the changes made in this PR.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Performance improvement
- [ ] Refactoring (no functional changes)

## Testing

- [ ] Tests added/updated for new functionality
- [ ] All existing tests pass
- [ ] Manual testing completed
- [ ] Edge cases tested

## Checklist

- [ ] Code follows the project's style guidelines
- [ ] Self-review of code completed
- [ ] Documentation updated (if applicable)
- [ ] No breaking changes (or breaking changes are documented)
- [ ] Changes are backward compatible
- [ ] Linting passes (`npm run lint`)
- [ ] Tests pass (`npm test`)
- [ ] Type checking passes (`npm run type-check`)

## Related Issues

Closes #(issue number)

## Screenshots (if applicable)

Add screenshots to help explain your changes.

## Additional Notes

Any additional information that reviewers should know.

### CODE REVIEW

File: packages/core/src/rules/require-test-assignee-comment.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Require an explicit assignee comment for Playwright `test.fixme` and `test.fail` cases.',
+      recommended: 'recommended',
+    },
+    schema: [],
+    messages: {
+      missingAssignee:
+        'test.fixme/test.fail must include an explicit assignee comment immediately above the call, e.g. "// @assignee @username".',
```
Comment (rvab) [inline packages/core/src/rules/require-test-assignee-comment.js:22]: * Should the comment include `@assignee` or just `assignee`?
* Will it work for multiline comment?

**Review by rvab — APPROVED** [review]

File: packages/core/src/rules/require-test-assignee-comment.js (hunk 1 above)
Comment (OmkarJ13) [reply packages/core/src/rules/require-test-assignee-comment.js:22]: Plus 1

**Review by OmkarJ13 — APPROVED** [review]
